### DISCUSSION TITLE
Upload failed due to Minio CORS Policy

### DISCUSSION THREAD

**Discussion by DarkdragonicuS** [outline/outline#5420] [Self Hosting]
Deployed Outline with Minio as storage. I can upload some files (guess, small sized), but trying to upload over 10 MB (as I can see) files failes with "Access to XMLHttpRequest at 'https://<minio_domain>/outline-bucket' from origin 'https://<outline_domain>' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource". In minio setting 'Cors Allow Origin' is set to '*', also added 'https://<outline_domain>'. I'm new to Minio, so can't figure out how to solve this issue.

**Comment by DarkdragonicuS** [comment]
It wasn't Outline problem. It was nginx proxy client_max_body_size option value too low.

**Comment by ovizii** [comment]
oops, wrong thread.
